src/test/jtx/TrustedPublisherServer.h

```diff
@@ -170,29 +176,77 @@ class TrustedPublisherServer
         auto const manifest = makeManifestString(
             publisherPublic_, publisherSecret_, keys.first, keys.second, 1);
 
-        std::string data = "{\"sequence\":" + std::to_string(sequence) +
-            ",\"expiration\":" +
-            std::to_string(expiration.time_since_epoch().count()) +
-            ",\"validators\":[";
+        std::vector<std::pair<std::string, std::string>> blobInfo;
+        blobInfo.reserve(futures.size() + 1);
+        auto const [data, blob] = [&]() {
```

Comment:
Consider a comment and adding a return type to the lambda. (I don't mind leaving off return types of most lamdas, but when the return type is not obvious and the lambda is more than a couple lines I think they improve the code).

Reply:
Done.